Lombiq.NodeJs.Extensions/pnpm-lock.yaml

```diff
@@ -1,8 +1,17 @@
 lockfileVersion: 5.3
 
 specifiers:
+  '@babel/cli': ^7.17.10
```

Comment:
I've seen this too, and have been unable to pinpoint the root cause 🤔

Reply:
Looks like [`core.safecrlf`](https://git-scm.com/docs/git-config#Documentation/git-config.txt-coresafecrlf) set to `false` would achieve what we need here.

But I am currently unable to test this, because the annoying change is somehow gone 😮

Reply:
Did you also close VS, clean the repo, and build after that?